lib/id3c/cli/redcap.py

```diff
@@ -69,6 +71,24 @@ def instruments(self) -> List[str]:
         return self._instruments
 
 
+    @property
+    def events(self) -> List[str]:
+        """
+        Names of all events in this REDCap project.
+        """
+        if self._events is None:
+            nameof = itemgetter("unique_event_name")
+
+            try:
+                self._events = list(map(nameof, self._fetch("event")))
+            except HTTPError:
+                LOG.debug("HTTP error encountered when fetching events. "
+                "Is this a classic project?")
+                self._events = []
```

Comment:
This will mask many kinds of real network errors unrelated to the availability of events in the project. Could we check the `is_longitudinal` field returned by the project info endpoint instead? It should already be in `self._details["is_longitudinal"]` and have a `"0"` or `"1"` value.

Reply:
Thanks, I wasn't a fan of this general HTTP Error exception, either. I thought I looked through all the fields for one matching "longitudinal", but I guess I didn't. I'll push a fix.

Reply:
Updated in the "Account for classic projects" commit.